internal/clients/diffreviewer/circleci/circleci_service.go

```diff
@@ -58,12 +80,56 @@ func (s *Service) LoadConfig(nightfallConfigFileName string) (*nightfallconfig.C
 		return nil, errors.New("missing env var for commit sha")
 	}
 	beforeCommitSha, _ := os.LookupEnv(CircleBeforeCommitEnvVar)
+
```

Comment:
can we clean up this super long function at all? Helper funcs and such at reasonable places?